src/Server/MySQLHandler.cpp

```diff
@@ -169,6 +169,11 @@ void MySQLHandler::run()
                 log->log(exc);
                 packet_sender->sendPacket(ERR_Packet(exc.code(), "00000", exc.message()), true);
             }
+            catch (const Poco::Exception & exc)
```

Comment:
And also `std::exception` and unknown exceptions?

PS. We can do it in more generic way: catch (...) and `getCurrentExceptionCode`, `getCurrentExceptionMessage`.

Reply:
Good, thanks for the advice, has been updated.